dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -241,6 +245,23 @@ internal constructor(
             return this
         }
 
+        /**
+         * Enable long operations on the main thread to be tracked automatically.
+         * Any long running operation on the main thread will appear as Long Tasks in Datadog
+         * RUM Explorer
+         * @param longTaskThresholdMs the threshold in milliseconds above which a task running on
+         * the Main thread [Looper] is considered as a long task (default 100ms)
+         */
+        @JvmOverloads
+        fun trackLongTasks(longTaskThresholdMs: Long = DEFAULT_LONG_TASK_THRESHOLD_MS): Builder {
```

Comment:
you are missing the equivalent for the deprecated `DatadogConfig.Builder` class

Reply:
`DatadogConfig.Builder` is deprecated. Not adding new features there is actually an incentive to switch to the new Configuration class.